generic/tests/netperf.py

```diff
@@ -482,6 +482,7 @@ def count_interrupt(name):
         sum = 0
         intr = []
         stat = ssh_cmd(server_ctl, "cat /proc/interrupts |grep %s" % name)
+        stat = stat.strip().split("\n")[-1]
```

Comment:
Not  exactly, the output very ugly like:
10:49:58 DEBUG| Sending command: echo $?
10:49:58 DEBUG| Sending command: cat /proc/interrupts |grep virtio.-input
10:49:58 DEBUG| Sending command: echo $?
10:49:58 DEBUG|  25:         54     162176   PCI-MSI-edge      virtio1-input.0
10:49:58 DEBUG|
10:49:58 DEBUG| Sending command: cat /proc/interrupts |grep virtio.-output
10:49:58 DEBUG| Sending command: echo $?
10:49:58 DEBUG| cat /proc/interrupts |grep virtio.-outpu<f-2.6.0]# cat /proc/interrupts |grep virtio.-output                         ^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H
10:49:58 DEBUG|  26:          2          0   PCI-MSI-edge      virtio1-output.0
10:49:58 DEBUG|

So the [-1] is needed and pass in my tests with x86 and ppc guest.